src/Microsoft.AspNetCore.Rewrite/Internal/IISUrlRewrite/UrlRewriteFileParser.cs

```diff
@@ -81,9 +81,9 @@ private void ParseRuleAttributes(XElement rule, UrlRewriteRuleBuilder builder)
             {
                 patternSyntax = PatternSyntax.ECMAScript;
             }
-            else if (!Enum.TryParse(rule.Attribute(RewriteTags.PatternSyntax).Value, ignoreCase: true, result:out patternSyntax))
+            else if (!Enum.TryParse(rule.Attribute(RewriteTags.PatternSyntax).Value, ignoreCase: true, result: out patternSyntax))
             {
-                ThrowParameterFormatException(rule, "The patternSyntax parameter was not recognized");
+                ThrowParameterFormatException(rule, $"The {RewriteTags.PatternSyntax} parameter: {rule.Attribute(RewriteTags.PatternSyntax).Value}, was not recognized");
```

Comment:
`parameter '{rule.Attribute(RewriteTags.PatternSyntax).Value}' was not recognized